skins/cat17/templates/Donation_Address_Update.twig

```diff
@@ -0,0 +1,26 @@
+{% extends 'Base_Layout.html.twig' %}
+
+{% block page_identifier %}address-update{% endblock %}
+
+{% block main %}
+	<div class="container">
+		<div class="row">
+			<div class="content col-xs-12 col-sm-10 col-md-8">
+				<h1>
+					{% if updateSuccessful %}
```

Comment:
I think moving the condition to the presenter and and assigning different translation keys there would make this template more readable.

It would also make the code more testable, as you could look for the translation keys in the output of the Edge2Edge test for the route and/or the Presenter Unit test.